libs/utils/utils.js

```diff
@@ -9,6 +9,15 @@ const AUTO_BLOCKS = [
   { fragment: '/fragments' },
 ];
 
+export function getEnvironment() {
+  const { hostname } = window.location;
+  if (hostname.includes('localhost')) return 'local';
+  /* c8 ignore start */
+  if (hostname.includes('hlx.page')) return 'draft';
```

Comment:
I probably wouldn't call this draft...

localhost == local
hlx.page || hlx.live == stage
else prod.